root/static/scripts/edit/URLCleanup.js

```diff
@@ -299,37 +308,38 @@ function disallow(url, id) {
  * - clean: Function to clean up/normalize matched URL.
  * - validate: Function to validate matched (clean) URL
  *             for an auto-selected relationship type.
- **/
+ *
+ */
 const CLEANUPS = {
-  wikipedia: {
-    match: [new RegExp("^(https?://)?(([^/]+\\.)?wikipedia|secure\\.wikimedia)\\.","i")],
+  'wikipedia': {
```

Comment:
What makes eslint add single quotes to `CLEANUPS` keys?

Reply:
Some keys were already quoted because they start with numbers, so I assume it's for consistency inside the same object